src/redux/movieSlice.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { SortingOptionsType } from "../types/sortingOptions.type";
+import { RootState } from "./store";
+
+const initialState = {
+  movieId: 0,
+  sortBy: "",
+  sortOrder: "",
+  filterBy: "",
```

Comment:
the user should be able to select multiple genres, not only comedy or only documentary but both of them